UI/CardViewMigration/CardViewMigration/CardViewMigration/Platforms/Android/Assets/AboutAssets.txt

```diff
@@ -4,7 +4,7 @@ in the shared project instead. Assets in this folder are Android-only assets.
 Any raw assets you want to be deployed with your application can be placed in
 this directory (and child directories) and given a Build Action of "AndroidAsset".
 
-These files will be deployed with your package and will be accessible using Android's
+These files will be deployed with you package and will be accessible using Android's
```

Comment:
Looks like "your" is the correct one here.

Reply:
@Youssef1313 Interesting, this change seems to be coming from the updated template as I didn't change anything here. Same issue is here for example: https://github.com/unoplatform/uno/blame/ad8a82d3ddeb7a70910ab563cccadd3a450c2ff5/src/SolutionTemplate/UnoAppWinUI/UnoAppWinUI.Mobile/Android/Assets/AboutAssets.txt#L7

Reply:
Yup, fixing in https://github.com/unoplatform/uno.templates/pull/899